signin/script.js

```diff
@@ -1,86 +1,71 @@
+  pwdInput,
+  signUpBtn,
+  toggleEye
+} from '../data.js';
 
+import { isValidEmail } from '../validation.js';
 
-const setError = (element, message) => {
-  const inputControl = element.parentElement;
-  const errorDisplay = inputControl.querySelector('.error');
 
-  errorDisplay.textContent = message;
-  inputControl.classList.add('error');
-  inputControl.classList.remove('success');
-}
-
-const setSuccess = element => {
-  const inputControl = element.parentElement;
-  const errorDisplay = inputControl.querySelector('.error');
-
-  errorDisplay.textContent = '';
-  inputControl.classList.add('success');
-  inputControl.classList.remove('error');
-}
-
-const isValidEmail = email => {
-  const regExp = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
-  return regExp.test(email);
-}
+const checkEmailValidation = (e) => {
+  const inputField = document.querySelector('.email-field');
+  const email = emailInput.value.trim();
+  const errorMessage = inputField.querySelector('.error');
 
-const checkEmailValidation = () => {
-  const emailValue = email.value.trim();
-  
-  if (emailValue === '') {
-    setError(email, '이메일을 입력하세요.');
-  } else if (!isValidEmail(emailValue)) {
-    setError(email, '올바른 이메일 주소가 아닙니다.')
+  if (email === '') {
+    errorMessage.textContent = '이메일을 입력해주세요.';
+    inputField.classList.add('error');
+  } else if (!isValidEmail(email)) {
+    errorMessage.textContent = '올바른 이메일 주소가 아닙니다.';
+    inputField.classList.add('error');
   } else {
-    setSuccess(email);
+    errorMessage.textContent = '';
+    inputField.classList.remove('error');
   }
-}
+};
 
-const checkPwdValidation = () => {
-  const passwordValue = password.value.trim();
-  
-  if (passwordValue === '') {
-    setError(password, '비밀번호를 입력하세요.');
+const checkPwdValidation = (e) => {
+  const inputField = document.querySelector('.pwd-field');
+  const pwd = pwdInput.value.trim();
+  const errorMessage = inputField.querySelector('.error');
+
+  if (pwd === '') {
+    errorMessage.textContent = '비밀번호를 입력하세요.';
+    inputField.classList.add('error');
   } else {
-    setSuccess(password);
+    errorMessage.textContent = '';
+    inputField.classList.remove('error');
   }
-}
+};
 
-const submit = () => {
-  const emailValue = email.value.trim();
-  const passwordValue = password.value.trim();
+
+
+const submit = (e) => {
+  e.preventDefault();
+
+  const email = emailInput.value.trim();
+  const password = pwdInput.value.trim();
   const testEmail = 'test@codeit.com';
   const testPassword = 'codeit101';
-
-  if (emailValue != testEmail && passwordValue 
-  != testPassword) {
-    setError(email, '이메일을 확인해주세요.');
-    setError(password, '비밀번호를 확인해주세요.');
-  } else if (emailValue != testEmail) {
-    setError(email, '이메일을 확인해주세요.');
-  } else if (passwordValue != testPassword) {
-    setError(password, '비밀번호를 확인해주세요.');
-  } else {
-    const link = '/folder.html';
-    location.href = link;
-  }
+  
+  if (email == testEmail && password == testPassword) {
```

Comment:
```suggestion
  if (email === testEmail && password === testPassword) {
```
문자열 비교라서 상관은 없지만 JS에서 값을 비교하실 때는 strict하게 해주시는 걸 추천드립니다 (==, === 의 차이도 한번 학습해보시면 좋을 거 같아요!).